[2025-04-14T16:19:19.362+0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-14T16:19:19.370+0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks.task_a manual__2025-04-14T13:19:15.701802+00:00 [queued]>
[2025-04-14T16:19:19.375+0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks.task_a manual__2025-04-14T13:19:15.701802+00:00 [queued]>
[2025-04-14T16:19:19.376+0300] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-14T16:19:19.391+0300] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task_a> on 2025-04-14 13:19:15.701802+00:00
[2025-04-14T16:19:19.395+0300] {standard_task_runner.py:72} INFO - Started process 73914 to run task
[2025-04-14T16:19:19.414+0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'executing_multiple_tasks', 'task_a', 'manual__2025-04-14T13:19:15.701802+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/execute_multiple_tasks.py', '--cfg-path', '/tmp/tmpuf2gzf29']
[2025-04-14T16:19:19.425+0300] {standard_task_runner.py:105} INFO - Job 28: Subtask task_a
[2025-04-14T16:19:19.726+0300] {task_command.py:467} INFO - Running <TaskInstance: executing_multiple_tasks.task_a manual__2025-04-14T13:19:15.701802+00:00 [running]> on host DESKTOP-1SBO1JU.
[2025-04-14T16:19:19.932+0300] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tasneemahmed' AIRFLOW_CTX_DAG_ID='executing_multiple_tasks' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2025-04-14T13:19:15.701802+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-14T13:19:15.701802+00:00'
[2025-04-14T16:19:19.934+0300] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-14T16:19:19.967+0300] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-14T16:19:19.969+0300] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Task A has Started!\n                            for i in {1..5}\n                            do\n                                echo "Task A is running iteration $i"\n                            done\n                                \n                            echo Task A has completed!']
[2025-04-14T16:19:19.983+0300] {subprocess.py:99} INFO - Output:
[2025-04-14T16:19:19.991+0300] {subprocess.py:106} INFO - Task A has Started!
[2025-04-14T16:19:19.995+0300] {subprocess.py:106} INFO - Task A is running iteration 1
[2025-04-14T16:19:19.996+0300] {subprocess.py:106} INFO - Task A is running iteration 2
[2025-04-14T16:19:19.996+0300] {subprocess.py:106} INFO - Task A is running iteration 3
[2025-04-14T16:19:19.996+0300] {subprocess.py:106} INFO - Task A is running iteration 4
[2025-04-14T16:19:19.996+0300] {subprocess.py:106} INFO - Task A is running iteration 5
[2025-04-14T16:19:19.997+0300] {subprocess.py:106} INFO - Task A has completed!
[2025-04-14T16:19:19.998+0300] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-14T16:19:20.025+0300] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-14T16:19:20.027+0300] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks, task_id=task_a, run_id=manual__2025-04-14T13:19:15.701802+00:00, execution_date=20250414T131915, start_date=20250414T131919, end_date=20250414T131920
[2025-04-14T16:19:20.079+0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-14T16:19:20.107+0300] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-04-14T16:19:20.117+0300] {local_task_job_runner.py:245} INFO - ::endgroup::
